test/json_logic_test.rb

```diff
@@ -70,4 +70,16 @@ def test_array_with_logic
       { "x" => true, "y" => 42}
     )
   end
+
+  def test_in_with_variable
```

Comment:
I'd like a negative test too please

Reply:
@mjuzsczak

Reply:
@mjuszczak

Reply:
@bhgames is this what you're looking for? https://github.com/JCotton1123/json-logic-ruby/pull/1/files

Reply:
Yes that would work, if you could get it in here.

Reply:
@eropple Can you merge in https://github.com/JCotton1123/json-logic-ruby/pull/1/files